### PR TITLE
chore: reorder automated archival steps

### PR DESCRIPTION
**Pull request by pepopowitz**
## Description

Part of #1173.

Reorders the automated archival steps, so that it's easier to open a PR with _just_ the massive deletion changes, followed by a PR for the more digestible changes.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: hacks/isolateVersion/allSteps.sh [hunk 1]
```diff
@@ -30,12 +30,12 @@ script_index=$1
...
-if [[ "$script_index" == 1 || -z "$script_index" ]]; then
-  source $script_directory/1-reduceVersionManifest.sh
+if [[ "$script_index" == 2 || -z "$script_index" ]]; then
+  source $script_directory/1-deleteOtherVersions.sh
 fi
 
-if [[ "$script_index" == 2 || -z "$script_index" ]]; then
-  source $script_directory/2-deleteOtherVersions.sh
+if [[ "$script_index" == 1 || -z "$script_index" ]]; then
+  source $script_directory/2-reduceVersionManifest.sh
```
Comment (akeller) [inline hacks/isolateVersion/allSteps.sh:38]: Is the index here right? The should be mismatched with the source?

Comment (pepopowitz) [reply hacks/isolateVersion/allSteps.sh:38]: I definitely screwed something up -- I'll fix!

Comment (akeller) [reply hacks/isolateVersion/allSteps.sh:38]: Just like I screwed up that second sentence 😆 Thank you for interpreting my gibberish.

Comment (pepopowitz) [reply hacks/isolateVersion/allSteps.sh]: fixed in latest commit!

**Review by akeller — APPROVED** [review]

**Review by akeller — APPROVED** [review]

**Pull request merged**